pkg/plugin/work-in-progress/plugin/event_handler_unit_test.go

```diff
@@ -1,8 +1,8 @@
 package plugin_test
 
 import (
+	"github.com/arquillian/ike-prow-plugins/pkg/internal/test"
```

Comment:
Minor thing, in `test-keeper/event_handler_gh_test.go` et al  we `.` import it - I think it would be good going one way or the other and in fact I think `test.` sounds a bit better. Can you align it?

Reply:
done aae7a75